packages/alfa-dom/src/node/element.ts

```diff
@@ -312,7 +319,7 @@ export class Element<N extends string = string>
       style: this._style.map((style) => style.toJSON()).getOr(null),
       shadow: this._shadow.map((shadow) => shadow.toJSON()).getOr(null),
       content: this._content.map((content) => content.toJSON()).getOr(null),
-      box: this._box.map((box) => box.toJSON()).getOr(null),
+      box: Option.from(options).flatMap(o => this._boxes.get(o.device)).map(box => box.toJSON()).getOr(null)
```

Comment:
I think the logic of checking the options for a device would be a bit clearer with optional chaining:
```suggestion
      box: options?.device === undefined ? null : this._boxes.get(options.device).map(box => box.toJSON()).getOr(null)
```